### PR TITLE
Updated for fixeddifficulty in genesis file

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated private network tutorial to use fixeddifficulty so it will mine blocks at a reasonable rate

### CODE REVIEW

File: docs/Tutorials/Create-Private-Network.md [hunk 1]
```diff
@@ -56,6 +57,7 @@ Copy the following genesis definition to a file called `privateNetworkGenesis.js
 {
   "config": {
```
Comment (NicolasMassart) [inline docs/Tutorials/Create-Private-Network.md]: Shouldn't the genesis contain a chain id parameter ? also if it has one, setting the network id in the cli would be useless as the default network id is the genesis chain id.

File: docs/Tutorials/Create-Private-Network.md [hunk 2]
```diff
@@ -56,6 +57,7 @@ Copy the following genesis definition to a file called `privateNetworkGenesis.js
...
     "ethash": {
+      "fixeddifficulty": 1000
```
Comment (NicolasMassart) [inline docs/Tutorials/Create-Private-Network.md]: Why 1000 ? why not 100 like in development network dev.json ?

File: docs/Tutorials/Create-Private-Network.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Tutorials/Create-Private-Network.md]: Updated

File: docs/Tutorials/Create-Private-Network.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Tutorials/Create-Private-Network.md]: As discussed - not quite as fast so you can see transactions going into the blocks.

**Pull request merged**